src/text-input/tests/unit/TextInput.spec.tsx

```diff
@@ -606,12 +610,12 @@ registerSuite('TextInput', {
 					css.noLabel
 				])
 				.prepend('@inputWrapper', () => [leading]);
-			const h = harness(() => <TextInput>{{ leading }}</TextInput>);
+			const h = harness(() => <TextInput>{{ leading: <span>A</span> }}</TextInput>);
```

Comment:
You could probably get away with just using a text node instead of a full span.
```suggestion
			const h = harness(() => <TextInput>{{ leading: 'A' }}</TextInput>);
```

Reply:
yup yup, you are correct thought wanted to show that you can add domnodes as well as text